spec/spec_helper.rb

```diff
@@ -2,6 +2,10 @@
 require "logstash/devutils/rspec/spec_helper"
 require 'logstash/inputs/udp'
 
+class LogStash::Inputs::Udp
+  attr_reader :udp
+end
```

Comment:
I am sure there is a valid reason for adding this here :P  - could you enlighten me and document it?